README.md

```diff
@@ -27,37 +27,55 @@ var clientsManager = RelationsGrpcClientsManager.forInsecureClients(url);
 var checkClient = clientsManager.getCheckClient();
 ```
 ### Making requests
-Synchronous:
+#### Synchronous example
+
+Let's say we want to check whether a subject of type "user" with id "bob" has "view" permission to a resource of type "workspace" and id "hosts". We can make the following synchronous check request:
```

Comment:
Should mention namespace as well to clarify why "rbac"

Reply:
I add a small amount of text, but wanted to avoid explanations here. See what you think.

Reply:
I like it, a good quick explanation that it exists in the "rbac" namspace 👍